passes/techmap/techmap.cc

```diff
@@ -1028,6 +1028,10 @@ struct TechmapPass : public Pass {
 		log("        map file. Note that the Verilog frontend is also called with the\n");
 		log("        '-nooverwrite' option set.\n");
 		log("\n");
+		log("    -dont_map <celltype>\n");
+		log("        ignore any mapping rules for the given cell type, that is leave it\n");
```

Comment:
Maybe re-order, something like "leave <celltype> unmapped by ignoring mapping rules for it"

Reply:
I've rephrased

Reply:
Thanks